core/iwasm/libraries/wasi-nn/lib_run_inference.cpp

```diff
@@ -4,20 +4,38 @@
 #include <tensorflow/lite/kernels/register.h>
 #include <tensorflow/lite/model.h>
 #include <tensorflow/lite/optional_debug_tools.h> 
+#include <tensorflow/lite/error_reporter.h> 
+
+enum Idx  {GRAPH=0, GRAPH_SIZE=1 };
+
 
 std::unique_ptr<tflite::Interpreter> interpreter = NULL;
 std::unique_ptr<tflite::FlatBufferModel> model = NULL;
 
-uint32_t _load(graph_builder_array builder, graph_encoding encoding) {
-	uint32_t *size = (uint32_t*) builder[1];
-	printf("inside _load: %d\n", *size);
-	return suces;
-    // tflite::ErrorReporter *error_reporter;
-    // model = tflite::FlatBufferModel::BuildFromBuffer(
-    //     (const char *)builder[0],
-    //     1000, // TODO: find how to pass buffer size
-    //     error_reporter
-    // );
-    // tflite::ops::builtin::BuiltinOpResolver resolver;
-	// tflite::InterpreterBuilder(*model, resolver)(&interpreter);
+uint32_t _load(graph_builder_array graph_builder, graph_encoding encoding) {
```

Comment:
According to https://github.com/WebAssembly/wasi-nn/blob/main/phases/ephemeral/witx/wasi_ephemeral_nn.witx#L110-L119, the parameter name is builder.